presto-raptor/src/main/java/com/facebook/presto/raptor/systemtables/ShardMetadataRecordCursor.java

```diff
@@ -328,6 +330,28 @@ else if (temporalType.equals(TIMESTAMP)) {
         return builder.build();
     }
 
+    static private void buildPredicatesAndValues(Domain nameDomain, List<String> values, List<String> predicates,
```

Comment:
This doesn't match our code style. All arguments should be on the same line, or each should be on a separate line. the continuation indent is 8 spaces (not aligned).